modules/template-parts/classes/render/header/render-contact-buttons.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+namespace HelloPlus\Modules\TemplateParts\Classes\Render\Header;
+
+use Elementor\Icons_Manager;
+use HelloPlus\Classes\Ehp_Social_Platforms;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+use HelloPlus\Classes\Render_Base;
+
+class Render_Contact_Buttons extends Render_Base {
+
+	public function render(): void {
```

Comment:
Possible we can flatten the structure:

```suggestion
public function render(): void {
		$buttons        = $this->settings['contact_buttons_repeater'];
		$link_type      = $this->settings['contact_buttons_link_type'];
		$display_mode   = $this->settings['contact_buttons_responsive_display'];
		$hover_animation = $this->settings['contact_button_hover_animation'];

		$this->widget->add_render_attribute( 'contact-buttons', [
			'class' => [
				$this->get_class_name( '__contact-buttons' ),
				'has-responsive-display-' . $display_mode,
			],
		] );

		$platforms = new Ehp_Social_Platforms( $this->widget );

		echo '<div ' . $this->widget->get_render_attribute_string( 'contact-buttons' ) . '>';

		foreach ( $buttons as $key => $button ) {
			$this->render_single_button( $platforms, $button, $key, $link_type, $hover_animation );
		}

		echo '</div>';
	}
```